README.md

```diff
@@ -106,6 +107,29 @@ used by setting `OCM_SERVICE_TOKEN` to your [OCM offline token](https://console.
 In OCM mode, it may take up to 10 minutes after the `kas-installer.sh` completes before the addon installations are ready. Until ready,
 the fleet-manager API will reject new Kafka requests.
 
+### Enterprise
+
+Enable by setting `ENTERPRISE_ENABLED='true'` either via kas-installer.env or the environment.
```

Comment:
The `OCM_SERVICE_TOKEN` needs to also be present. Otherwise, KFM reconciliation will `panic` :-)